clients/fides-js/src/lib/overlay.module.css

```diff
@@ -5,7 +5,8 @@
   /* Colors */
   --fides-consent-overlay-primary-color: #8243f2;
   --fides-consent-overlay-background-color: #f7fafc;
-  --fides-consent-overlay-font-color: #4a5568;
+  --fides-consent-overlay-font-color-light: #4a5568;
```

Comment:
nit: can we keep this as `--fides-consent-overlay-font-color` and then have the other one just be `--fides-consent-overlay-font-color-dark` or even `--fides-consent-overlay-link-color`? `-light` makes me think it'll be light, not a body color 🙂

Reply:
yes good idea, thanks for the review!